### ISSUE TITLE
FR: Add divider to show tabs grouped by workspace splits

### ISSUE THREAD

**Issue by gasparschott** [hdykokd/obsidian-vertical-tabs-view#84]
This would improve clarity.
Also, perhaps add styling to wrap long file names and expand height of individual list items to fit.

**Comment by uwe999** [comment]
you would just create notes called something like "------"

**Comment by gasparschott** [comment]
True, but this would have to be done every time the split layout changes, wouldn’t it? And it seems rather messy to have to add notes just for this purpose. I assume your plugin iterates through all the open leaves to create the list; wouldn’t it be possible to add a “first” class to the li corresponding to the first leaf in each split? Thanks.

**Comment by uwe999** [comment]
not my plugin, I just tried to find a workaround. I'm now using Obsidian's native bookmark feature - it has a grouping feature and it's easy to rearrange the list

**Comment by wwjCMP** [comment]
I think this should be implemented, a very practical improvement

**Comment by hdykokd** [comment]
I once tried to implement this, but due to the current lack of API support for Obsidian's Workspace Layout, it is difficult to implement synchronization.
For this reason, the Vertical Tabs View manages the tabs independently.
It is possible to rearrange the tabs by dynamically generating and saving the Workspace layout, but every time the Workspace layout is saved, the entire screen reloads (causing flickering), which is not acceptable for me.
I would like to wait for Obsidian to implement the necessary API.
